doc/articles/how-uno-works.md

```diff
@@ -2,60 +2,127 @@
+
+Skia is the default rendering engine for Uno Platform as of Uno Platform 6.0 or later across **Desktop (Windows, macOS, Linux)**, **WebAssembly (WASM)**, **Android**, and **iOS**.
+
+> [!IMPORTANT]
+> **WinAppSDK** continues to use the official native rendering provided by Microsoft, as Uno Platform rendering is not present for this target.
 
-The rest of this article discusses how the Uno.UI tooling allows WinUI-compatible XAML and/or C# applications to run on non-Windows platforms.
+It provides a **consistent, pixel-perfect rendering pipeline** by drawing the entire UI using the Skia graphics library (via SkiaSharp).
 
-## Uno.UI at runtime
+The Skia Rendering backend has a very cheap cost for creating UI elements, which makes it very efficient for large user interfaces.
 
-The [`Uno.WinUI` NuGet package](https://www.nuget.org/packages/Uno.WinUI/) completely reproduces the WinUI API surface: all namespaces (`Microsoft.UI.Xaml`, `Windows.Foundation`, `Windows.Storage`, etc), all classes, all class members. Insofar as possible, the same look and behavior as on Windows are replicated on all other platforms.
+#### Supported Platforms
 
-Note that, as the API surface is very large, some parts are included but not implemented. These features are marked with the `Uno.NotImplementedAttribute` attribute, and a code analyzer is included with the Uno.WinUI package will generate a warning for any such features that are referenced. You can see a complete list of supported APIs [here](implemented-views.md).
+Skia rendering is available on:
 
-## How the UI is rendered
+- macOS
+- Linux
+- Windows
+- WebAssembly
+- Android
+- iOS
 
-As an application developer, you normally don't need to worry about exactly how Uno renders your visual tree to the screen on each platform. For every platform, the public API - panels, control templates, measurement and arranging, etc - is the same. However, it's sometimes useful to know what's going on at render-time, for example if you're [intermixing native views with XAML views](xref:Uno.Development.NativeViews) on a platform-specific basis.
+#### How Skia Rendering Works
 
-### Web (WebAssembly)
+- The entire UI is drawn on a Skia canvas
+- There are **no native views**; all visuals are composed in Skia using vector graphics
+- A minimal native shell (like a window or web canvas) hosts the Skia surface
 
-On the web, each XAML element is converted into an appropriate [HTML element](https://developer.mozilla.org/en-US/docs/Glossary/Element). Panels, controls, and other 'intermediate' elements in the visual tree are converted to `<div/>` elements, whereas 'leaf' elements like `TextBlock`, `Image` etc get converted into more specific tags (`<p/>`, `<img/>` etc).
+> [!NOTE]
+> Because it bypasses native UI components, Skia can offer pixel-perfect rendering and visual consistency. The same UI offered by default, but platform specific theming is possible using [Uno.Themes](xref:Uno.Themes.Overview).
```

Comment:
```suggestion
> Because it bypasses native UI components, Skia can offer pixel-perfect rendering and visual consistency. The same UI is offered by default, but platform-specific theming is possible using [Uno.Themes](xref:Uno.Themes.Overview).
```